mobile/src/common/enums/colors/colors.ts

```diff
@@ -0,0 +1,5 @@
+export { BackgroundColor } from './background-color.enum';
```

Comment:
Let's keep all colors in one enum - AppColor and make it nested like in common/enums/app/env.enum.ts. Also rename the folder to ui

Reply:
> Let's keep all colors in one enum - AppColor and make it nested like in common/enums/app/env.enum.ts. Also rename the folder to ui

Is color naming ok, or should be changed?